test/e2e_tsickle_compiler_host_test.ts

```diff
@@ -85,7 +84,7 @@ describe('tsickle compiler host', () => {
 
   it('lowers decorators to annotations', () => {
     const [program, compilerHost, options] =
-        makeProgram('foo.ts', '/** @Annotation */ const A: Function = null; @A class B {}');
+        makeProgram('foo.ts', '/** @Annotation */ function A(t: any) {return t}; @A class B {}');
```

Comment:
For my information, is this just a cleanup you noticed, or do we type check the program now and didn't before? In either case, a reasonably workaround for "I need an annotation and cannot be bothered to write its gnarly type" is also `declare const A: any;`.